tasks/mod-php.yml

```diff
@@ -2,7 +2,7 @@
 - name: ensure timezone is set in apache2 php.ini
   lineinfile: dest=/etc/php5/apache2/php.ini
               regexp='date.timezone ='
-              line='date.timezone = {{ server.timezone }}'
+              line='date.timezone = {{ php.timezone }}'
```

Comment:
For now don't change this... just ad a server.timezone in the defaults....

I know.. not cute but I want to make the change in phansible transparent.
And this kind of changes will require modifying the options in phansible..

Create an issue for it and we will take care when phansible starts using this roles.